SETUP/upgrade/06/update_projects_table.php

```diff
@@ -63,8 +63,7 @@
     FROM projects
     WHERE LEFT(comments,8) = 'special:'
 ") or die(mysqli_error(DPDatabase::get_connection()));
-$row = mysqli_fetch_row($res);
-$num_left = $row[0];
```

Comment:
do we in general want to update the upgrade scripts? It just increases test scope that I am not sure is entirely necessary.

Reply:
Perhaps not. I'm happy to leave them be if you prefer.

Reply:
This upgrade script in particular is going to get blown away in https://github.com/DistributedProofreaders/dproofreaders/pull/458 so I don't care either way and am happy to punt that decision down the road a bit.